crates/virtio-queue/src/descriptor.rs

```diff
@@ -113,7 +113,6 @@ impl Descriptor {
     }
 }
 
-#[cfg(any(test, feature = "test-utils"))]
```

Comment:
This should stay as before, and instead mark the test where it is needed with the `test-utilities` feature.

```rust
#[test]
#[cfg(feature = "test-utils")]
fn test_invalid_avail_idx() {
     ....
}
```

Reply:
I tried this, but I receive this error: 
error[E0599]: no function or associated item named `new` found for struct `Descriptor` in the current scope
   --> crates/virtio-queue/src/state.rs:171:36. I added #[cfg(feature = "test-utils")] for both test_invalid_avail_idx from queue.rs and test_queue_guard_object from queue_guard.rs